org.jitsi.jitsi-meet.appdata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```

Comment:
This should be upstream.

Reply:
Do you mean a literal `"upstream"` type? Considering the line it's a comment on.
Or that the entire appdata file should be from upstream? (They don't generate one, and have so far not wanted to accept any changes to their code for AppImage-unrelated things)

Reply:
Except appstream is not a Flatpak specific thing but a freedesktop standard.

Reply:
Yep, and technically AppImages require an appstream file too, but the electron-builder utility seems to ignore that requirement.
And seeing as Jitsi Meet is using electron-builder to build their AppImages, with no special configuration or additional files, only packing what is generated by that tool...

Reply:
meh, electron-builder isn't doing Linux users any good